src/MicrosoftTeams.ts

```diff
@@ -463,7 +465,12 @@ namespace microsoftTeams {
     ensureInitialized();
 
     let messageId = sendMessageRequest(parentWindow, "getContext");
-    callbacks[messageId] = callback;
+    callbacks[messageId] = (context: Context) => {
+      if (hostClientType != null) {
```

Comment:
This is a nice stop-gap but eventually it might be best if the Teams client just added this to the context

Reply:
Mobile Teams client has added in context too.
This was done to support hostClientType on web/desktop too. In case, web/desktop can also add in the context we don't need this.

Reply:
done